Yosemite/Yosemite/Tools/POS/POSOrderService.swift

```diff
@@ -58,6 +63,17 @@ public final class POSOrderService: POSOrderServiceProtocol {
         }
         return syncedOrder
     }
+
+    public func sendOrderReceipt(order: Order, toEmailAddress: String) async throws {
+        guard order.billingAddress?.email == nil || order.billingAddress?.email == "" else {
+            throw NSError(domain: "Email already set", code: 0)
```

Comment:
It would be good to define an `Error` enum for this, rather than using NSError.

Reply:
True! no reason to leave it for later. Changed: https://github.com/woocommerce/woocommerce-ios/pull/14481/commits/952aef8f9d17072051292482c69e599b719bd909